app/javascript/src/Navigation/components/header/SessionMenu.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Avatar } from '@patternfly/react-core'
+import avatarImg from 'Navigation/images/img_avatar.svg'
+
+const SessionMenu = ({avatarLinkClass, impersonated, accountName, displayName, logoutPath, username}) => (
+  <div className="PopNavigation PopNavigation--session">
+    <a className={`PopNavigation-trigger u-toggler ${avatarLinkClass} pf-m-avatar-link`} href="#session-menu" title="Session">
+      <Avatar src={avatarImg} className="pf-m-avatar-img"/>
+      {impersonated ? <i className="fa fa-bolt "></i> : null }
+    </a>
+    <ul className="PopNavigation-list u-toggleable" id="session-menu">
+      <li className="PopNavigation-listItem">
+        <p className="PopNavigation-info">
+          { impersonated ? <i className="fa fa-bolt "></i> : null }
+          { impersonated ? ' Impersonating a virtual admin user from ' : 'Signed in to '}
```

Comment:
Maybe these 2 lines could become:
```jsx
{ impersonated ? <i className="fa fa-bolt "></i>'Impersonating a virtual admin user from ' : 'Signed in to ' }
```

But what is better here is subjective and anyway I wouldn't ask you to repush only for this

Reply:
It should be like @Martouta said